new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -0,0 +1,385 @@
+  private static String domain2Namespace = null;
+
+  // domain constants
+  private final String domainUid = "domain1";
+  private final String clusterName = "cluster-1";
+  private final int replicaCount = 2;
+  private final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private final String managedServerPodPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+
+  // operator constants
+  private static HelmParams opHelmParams;
+  private static String opServiceAccount;
+  private static final String domainNamespaceSelectionStrategy = "Dedicated";
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and domain2.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism.
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // in the dedicated mode, the operator only manages domains in the operator's own namespace
+    domain1Namespace = opNamespace;
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain2Namespace = namespaces.get(1);
+
+    // Variables for Operator
+    opServiceAccount = opNamespace + "-sa";
+    opHelmParams =
+        new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+  }
+
+  /**
+   * When installing the Operator via helm install,
+   * set the Operator Helm Chart parameter domainNamespaceSelectionStrategy to Dedicated and
+   * set domainNamespaces to something that is different from the operator's namespace.
+   * Make sure that the domains is not in the operator's target namespaces do not come up.
+   *   Install an Operator with a namespace and set domainNamespaces in a different namespace
+   *     from the Operator's namespace, also set domainNamespaceSelectionStrategy to Dedicated
+   *     for the Operator Helm Chart.
+   *   Verify the Operator is up and running.
+   *   Create WebLogic Domain in a namespace that is different from the Operator's namespace.
+   *   Verify that the domain does not come up.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Set domainNamespaceSelectionStrategy to Dedicated for the Operator Helm Chart and "
+      + "verify that a domain not deployed in operator's namespace doesn't come up")
+  public void testDedicatedModeDiffNamespace() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa",
```

Comment:
are you using domainNamespace in this method when domainNamespaceSelectionStrategy is set?

Reply:
in dedicated mode, any value for domainNamespaces is ignored, we can pass empty or anything, the Operator will ignore it. The reason the test gives domain2Namespace is for another test case, just want to install the operator once and test all